app/views/components/FooterLinks.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components
+
+import config.AppConfig
+import controllers.routes
+import javax.inject.Inject
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.footer.FooterItem
+
+class FooterLinks @Inject() (implicit appConfig: AppConfig) {
+
+  def accessibilityLink()(implicit messages: Messages): FooterItem = FooterItem(
+    Some(messages("footer.links.accessibility.text")),
```

Comment:
footer.links.accessibility.text is missing from messages file, link doesnt render correctly

Reply:
new messages file pushed